app/source/components/card/card.scss

```diff
@@ -7,19 +7,14 @@
   );
   border-radius: 20px;
   box-shadow: 0 20px 50px rgba(black, 0.1);
-  transition: background 0.5s;
+  transition: background-color 0.5s;
 
   &.theme-red {
     background-color: $red;
   }
 
   &.theme-yellow {
     background-color: $yellow;
-    background-image: linear-gradient(
```

Comment:
If I remember correctly it's so that the transition looks better.

Also: it just looked like noise, and I didn't see the point of having it there. `// NOTE:` it next time 😛.

Reply:
I'm left wondering how I should know what would look like noise to you 🤔
Anyhoo, if it fixes a problem with the transitions that's cool